kokoro/gcp_ubuntu/releases/kokoro_build_release.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# A Kokoro build script to build release and update github repo
+
+# Code under repo is checked out to ${KOKORO_ARTIFACTS_DIR}/github.
+# The final directory name in this path is determined by the scm name specified
+# in the job configuration.
+
+# Display commands being run.
+# WARNING: please only enable 'set -x' if necessary for debugging, and be very
+#  careful if you handle credentials (e.g. from Keystore) with 'set -x':
+#  statements like "export VAR=$(cat /tmp/keystore/credentials)" will result in
+#  the credentials being printed in build logs.
+#  Additionally, recursive invocation with credentials as command-line
+#  parameters, will print the full command, with credentials, in the build logs.
+# set -x
+# Fail on any error.
+set -e
+
+#Variables
+export GIT_PSW="$(<${KOKORO_KEYSTORE_DIR}/76474_bqassessment-github-releases-02032022)"
+export GIT_RELEASES_USERNAME="edw-assessment-integration-testing-bot"
+export KOKORO_BUILD_RELEASE_DIR="${KOKORO_ARTIFACTS_DIR}/github/dwh-assessment-extraction-tool"
+export KOKORO_RELEASE_OUTPUT_FILE="${KOKORO_BUILD_RELEASE_DIR}/bazel-bin/dist/dwh-assessment-extraction-tool.zip"
+export SCRIPT_DIR=$(dirname "$0")
+export SCRIPT_PARENT_DIR="$(dirname -- "$SCRIPT_DIR")"
+export BUILD_SCRIPT="$SCRIPT_PARENT_DIR/kokoro_build.sh"
+source ${SCRIPT_DIR}/release_utils.sh
+
+cd ${KOKORO_BUILD_RELEASE_DIR}
```

Comment:
Please enclose dynamic arguments with double quotes to prevent issues with spaces in the variables.